cmd/minikube/cmd/start.go

```diff
@@ -173,6 +173,15 @@ func runStart(cmd *cobra.Command, args []string) {
 		os.Setenv(k, v)
 	}
 
+	//Create the host directories if not present
+	hostDirs := []string{viper.GetString(hostConfigDir), viper.GetString(hostDataDir), viper.GetString(hostVolumesDir)}
+	err = cluster.EnsureHostDirectories(libMachineClient, hostDirs)
```

Comment:
This is more and more entangling the CLI command line parsing code with the _cluster up_ code. I would love to see issue #409 addressed at some stage.